translations/en/LC_MESSAGES/messages.po

```diff
@@ -939,6 +945,9 @@ msgstr "Link"
 msgid "list"
 msgstr "a list"
 
+msgid "list_variable_role"
+msgstr ""
```

Comment:
Hi @Julia! Now what you have to do is type the meaning of this label inside `msgstr`. Then you need to do `doit run devserver` and this will compile the labels and you should see the appropiate strings afterwards.

Reply:
I did that, but it did not work... Somehow the msgstr things don't stay there. I put it in the messages.pot file, is that not the right one? Should I put it in the English file messages.po ?

Reply:
Yes, precisely, you should add it in the message.po file inside the `en` folder.